### PR TITLE
DDF-1714: Updated OWASP Issues

### PR DESCRIPTION
**Pull request by AzGoalie**
@lessarderic @brendan-hofmann @spearskw 
Needed for DDF-1714 https://github.com/codice/ddf/pull/380

### CODE REVIEW

**Comment by brendan-hofmann** [comment]
Do we have any way to know if we accidentally introduce a suppressed test/documentation vulnerability into production code some day? Some of the suppressions seem broad enough they could accidentally hide real problems someday.

**Comment by brendan-hofmann** [comment]
:+1:

**Comment by vinamartin** [comment]
@brendan-hofmann One way to make sure that real vulnerabilities aren't suppressed is to be more explicit with the suppression. For example, noting the specific jar with the CVE # in the suppression. That way if the same CVE comes up again for a different jar that is a true one, it can be checked. However, this also leads to a lot of false positives.

File: support-owasp/src/main/resources/dependency-check-maven-config.xml [hunk 1]
```diff
@@ -76,4 +76,40 @@
...
+    <!--
+        CVE-2007-1085 is a cross-site scripting (XSS) vulnerability related to Google Desktop
+        which does not apply here.
+    -->
+    <suppress>
+        <cve>CVE-2007-1085</cve>
+    </suppress>
+
+    <!--
+        CVE-2014-0003, CVE-2014-0002 Vulnerabilities in camel before 2.12.3, we use 2.14.2, false positive?
```
Comment (lessarderic) [inline support-owasp/src/main/resources/dependency-check-maven-config.xml]: May want to remove "false positive?" from the comment.

**Comment by lessarderic** [comment]
:+1:

**Pull request merged**